src/main/java/org/aksw/simba/lemming/creation/PersonGraphDataset.java

```diff
@@ -22,23 +23,31 @@ public PersonGraphDataset() {
 	
 	@Override
 	public ColouredGraph[] readGraphsFromFiles(String dataFolderPath) {
-		 
+		
 		 List<ColouredGraph> graphs = new ArrayList<ColouredGraph>();
-		 GraphCreator creator = new GraphCreator();
+		 GraphCreator creator = new GraphCreator();		
 		 
 		 File folder = new File(dataFolderPath);
 		 if(folder != null && folder.isDirectory() && folder.listFiles().length > 0){
 			 List<String> lstSortedFilesByName = Arrays.asList(folder.list());
 			 //sort ascendently
-			 Collections.sort(lstSortedFilesByName);
+			 Collections.sort(lstSortedFilesByName);		
+			 
 			 for (String fileName : lstSortedFilesByName) {
 				 File file = new File(dataFolderPath+"/"+fileName);
 				 
 				 if(file != null && file.isFile() && file.getTotalSpace() > 0){
 					 Model personModel = ModelFactory.createDefaultModel();
 					 //read file to model
 					 personModel.read(file.getAbsolutePath(), "TTL");
-					 LOGGER.info("Read data to model - "+ personModel.size() + " triples");
+					 LOGGER.info("Read data to model - "+ personModel.size() + " triples");			 
+					 
+					 Inferer inferer = new Inferer();
+					 String ontFilePath = "dbpedia_2015-04.owl";
+					 OntModel ontModel = inferer.readOntology(ontFilePath);
+					//returns a new model with the added triples
+					 //Model newModel = 
+					 inferer.process(personModel, ontModel);
```

Comment:
Shouldn't this line be `personModel = inferer.process(personModel, ontModel);` ?